acceptable/__main__.py

```diff
@@ -233,31 +233,32 @@ def render_markdown(metadata, cli_args):
         for api in apis.values():
             # collect global changelog
             for changed_version, log in api.get('changelog', {}).items():
-                changelog[changed_version][api['api_name']] = log
+                changelog[changed_version][(group, api['api_name'])] = log
 
-        any_documented = any(
-            not api.get('undocumented', False) for api in apis.values()
+        documented_apis = OrderedDict(
+            (name, api) for name, api in apis.items()
+            if not api.get('undocumented', False)
```

Comment:
ISTM that you're not using the keys of this dict, so this could just be a list e.g.
```
documented_apis = [api for api in apis.values if not api.get('undocumented', False)]
```

Reply:
So it could, will fix